src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -59,21 +59,23 @@ interface ConfirmBidProps extends ReactStripeElements.InjectedStripeProps {
 const MAX_POLL_ATTEMPTS = 20
 
 export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
+  let timer = null
```

Comment:
I'm thinking this is okay since these variables (timer, pollcount) aren't used to render the form, but thought I'd mention anyways - is it okay in react to be declaring and mutating objects in the closure of a FC? I'm thinking yes, no need to use a state hook, but wanted to mention it.